tests/test_debug.py

```
@@ -0,0 +1,81 @@
+"""Tests for debug.py module."""
+
+import logging
+
+import pytest
+
+from kubeseal_auto.debug import configure_debug, ic, logger
+
+
+@pytest.fixture(autouse=True)
+def _reset_logger():
```

Ruff (0.15.9)

[warning] Missing return type annotation for private function `_reset_logger`

(ANN202)